gpu-service/main.py

```
@@ -0,0 +1,145 @@
+            'matches': results, 
+            'faces_found': len(face_encodings),
+            'face_locations': face_locations
+        })
+
+    except Exception as e:
+        logger.error(f"Lookup error: {e}")
+        return jsonify({'error': str(e)}), 500
+
+if __name__ == '__main__':
+    logger.info("Starting Flask app with lookup endpoint only...")
+    app.run(host='0.0.0.0', port=5001) 
```

Ruff (0.11.9)

Possible binding to all interfaces

(S104)